source/includes/administration/_commitments.md

```diff
@@ -112,6 +116,7 @@ Attributes | &nbsp;
 `status`<br/>*enum* | The status of the commitment. Possible values are: `IN_PROGRESS`, `UPCOMING`, and `EXPIRED`.
 `type`<br/>*enum* | The type of the commitment. Currently limited to only `RESOURCE`.
 `fixedPrice`<br/>*double* | The effective price for this commitment. Applies only to commitments with pricing method `FIXED_PRICE`.
+`fixedPriceTaxCode`<br/>*double* | The chosen tax code for this commitment. Applies only to commitments with pricing method `FIXED_PRICE`.
```

Comment:
```suggestion
`fixedPriceTaxCode`<br/>*TaxCode* | The chosen tax code for this commitment. Applies only to commitments with pricing method `FIXED_PRICE`.
```

Reply:
Can we add a small section below for a `TaxCode` object? Similarly to what we do for `Organization` and `CommittedProduct`

Reply:
Same for below 👇